### ISSUE TITLE
`Cards in navigable grid` example is missing position narration 

### ISSUE THREAD

**Issue by kolaps33** [microsoft/fluentui#14111]
This ticket is output of the user testing.

**Example:  Cards in navigable grid**

Repro steps:
1. Navigate into the card in the example
2. Observe narration

Actual: position card in the cards grid is not narrted
Expected: position is narrated
